.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+<!-- describe the changes you PR introduces here -->
+
+# Issues
+
+<!-- link related issues here -->
+
+# Checklist
+
+<!-- take care of the default items before creating your PR and be prepared to add more items -->
+
+* [ ] I updated the version in Chart.yaml
+* [ ] I added to the docs and updated them using  `pre-commit run -a`
```

Comment:
```suggestion
* [ ] I added the changes to the docs and updated them using  `pre-commit run -a`
```

Reply:
We have not been adding changes to the docs so far (as in a changelog per chart) so I'd like to use a different wording here.

Reply:
okay. But is the phrase "I added to the docs" correct? seems wrong to me. But english is not my native lang.

Reply:
i don't like current phrasing either, i'll try to come up with a better one

Reply:
```suggestion
* [ ] I updated the charts README.md using `pre-commit run -a`
```
The structure of the comments in `values.yaml` should probably be documented in `CONTRIBUTING.md`.

Reply:
the "adding to the docs" part tried to allude to the existence of the `docs/` folder and the fact that we should not forget to document high-level concepts in there, i might split this into two bullets.